controllers/limitador_controller_test.go

```diff
@@ -143,6 +153,27 @@ var _ = Describe("Limitador controller", func() {
 			}, timeout, interval).Should(Equal("http://" + limitadorObj.Name + ".default.svc.cluster.local:8000"))
 
 		})
+		It("Should create a ConfigMap with the correct limits and hash", func() {
+			createdConfigMap := v1.ConfigMap{}
+			Eventually(func() bool {
+				err := k8sClient.Get(
+					context.TODO(),
+					types.NamespacedName{
+						Namespace: LimitadorNamespace,
+						Name:      limitador.LimitsCMNamePrefix + limitadorObj.Name,
+					},
+					&createdConfigMap)
+
+				return err == nil
+			}, timeout, interval).Should(BeTrue())
+
+			Expect(createdConfigMap.Data[limitador.LimitadorCMHash]).Should(
+				Equal("a00c9940ae6bb8de702633ce453e6a97"),
+			)
+			Expect(createdConfigMap.Data[limitador.LimitadorConfigFileName]).Should(
+				Equal("- conditions:\n  - req.method == GET\n  max_value: 10\n  namespace: test-namespace\n  seconds: 60\n  variables:\n  - user_id\n- conditions:\n  - req.method == POST\n  max_value: 5\n  namespace: test-namespace\n  seconds: 60\n  variables:\n  - user_id\n"),
```

Comment:
what about `yaml.Marshal(limits)`?

Reply:
Funny tho, I started that way and then changed it to this more explicit approach. Comparing to `yaml.Marshal(limits)` makes it tidier and it's not difficult to understand the assertion, will change.

Reply:
I think it's not tidier in the end, since `Marshal` method also returns `error` so better leave it explicit as it is, no?

Reply:
ok. Just for curiosity; what is the issue that Marshal returns an error? You can always add assertion that the error is nil

Reply:
No issue at all, just the extra step of nil assertion